inflection/tools/dictionary-parser/src/main/java/org/unicode/wikidata/Inflection.java

```diff
@@ -1,24 +1,24 @@
 /*
+ * Copyright 2025 Unicode Incorporated and others. All rights reserved.
  * Copyright 2014-2024 Apple Inc. All rights reserved.
  */
-package com.apple.lexicon;
+package org.unicode.wikidata;
 
 import javax.annotation.Nonnull;
 import java.util.Comparator;
 import java.util.Iterator;
 import java.util.Collection;
 import java.util.TreeSet;
-import static com.apple.lexicon.Grammar.PartOfSpeech;
-import static com.apple.lexicon.Grammar.LemmaCategory;
-import static com.apple.lexicon.Grammar.Miscellaneous;
+import static org.unicode.wikidata.Grammar.Ignorable;
+import static org.unicode.wikidata.Grammar.PartOfSpeech;
 
 
 /**
  * @author George Rhoten 2014/11/20
  */
```

Comment:
I would like a more detailed comment on all complex Java classes.

E.g. Inflection class generates X from Y etc.

Reply:
Can you elaborate on what you're looking for?  Is there another way to state your example?

Reply:
I've added some documentation to the classes. Perhaps that is what you're looking for?

Reply:
Yes, thank you.